src/decode.h

```diff
@@ -240,6 +240,8 @@ typedef struct {
 
     /* instruction length */
     uint8_t insn_len;
+    /* To determine whether the IR is last IR in a basic block */
```

Comment:
More comments for the idea behind tail call optimizations.

Reply:
You can take some wording in [tails](https://github.com/snej/tails) which would describe how TCO works.